storage.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"io"
+	"net/http"
+)
+
+func handleIndex(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Served-With-Love-By", "Gophers")
+
+	bucket := client.Bucket(bucketName)
```

Comment:
i don't even think you need this! you could just read the index.html from local file system :D 

so we have 3 other html pages we need to serve also, invite.html, failure.html and success.html
but you can just serve them just like you're doing below with the assets.

you'd still need to do the path manipulation you're doing below, b/c like "/" == "/index.html"
but the rest of them should resolve themselves, so you would have the dir be "." or wherever the html is, and then "/invite.html" would just word :D

lmk if that makes sense..